exercises/practice/allergies/.meta/example.awk

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env gawk -f
+
+@include "join"
+
+BEGIN {
+    FS = ","
```

Comment:
Many of the other exercises use space separated words. Do we want to be consistent?

Reply:
I think we want to show many different file formats to be processed.

I'm thinking that the ETL exercise needs to be written to process more than a single line at a time